src/apps/wallet/sign_tx/scripts.py

```diff
@@ -158,31 +158,38 @@ def get_p2wsh_witness(multisig: MultisigRedeemScriptType, signature: bytes, sign
     for s in signatures:
         append_signature(w, s, sighash)  # size of the witness included
 
-    redeem_script = script_multisig(multisig_get_pubkeys(multisig), multisig.m)
-
+    # redeem script
+    pubkeys = multisig_get_pubkeys(multisig)
+    redeem_script = script_multisig(pubkeys, multisig.m)
     write_varint(w, len(redeem_script))
     write_bytes(w, redeem_script)
     return w
 
 
 # -------------------------- Multisig --------------------------
 
-def input_script_multisig(current_signature, other_signatures, pubkeys, m: int, sighash: int):
+def input_script_multisig(multisig: MultisigRedeemScriptType, signature: bytes, signature_index: int, sighash: int):
+
+    signatures = multisig.signatures  # other signatures
+    if len(signatures[signature_index]) > 0:
```

Comment:
Tohle nevyhodí error, že to pole takový index nemá? Jakože len() vyhodí 0 pokud to zavoláš na neexistujícím prvku místo error?

Reply:
Beru zpět, už to chápu. To pole ten prvek má a obsahuje `b''`, takže je to ok. :+1: